src/config/dynamic-theme-fixes.config

```diff
@@ -24604,6 +24604,13 @@ INVERT
 
 ================================
 
+vowi.fsinf.at
+
+INVERT
+img
```

Comment:
INVERT for all images can break other images on this site in other pages.

As it seems to be wikipedia clone page i propose to add to INVERT section:
```
.mwe-math-fallback-image-inline
.mwe-math-fallback-image-display
```